### ISSUE TITLE
doesNotHaveStyle assertion

### ISSUE THREAD

**Issue by timiyay** [simplabs/qunit-dom#449]
I recently needed a `doesNotHaveStyle` assertion, as a counterpart to test that uses the existing `hasStyle` assertion: https://github.com/simplabs/qunit-dom/blob/master/API.md#hasstyle

Would this feature be considered, if I opened a PR?

Here's what I'm doing right now, within an EmberJS component rendering test:

```js
const element = this.element.querySelector('[data-test-my-component]')
const zIndex = window.getComputedStyle(element).zIndex
assert.notEqual(zIndex, '-1', 'should have non-negative z-index')
```

**Comment by Turbo87** [comment]
seems exotic, but if it's reasonably easy to implement we can consider it :)

**Comment by timiyay** [comment]
Thanks for the quick reply @Turbo87. This task will go on my "20% time open-source source warm fuzzy feeling list", which gets looked at on Friday afternoons.

I'll get a PR going, and we can decide from there whether it's worthy merging or not.

**Comment by scalvert** [comment]
I hope you didn't start working on this yet, @timiyay. I had some spare cycles, so tossed it together.

**Comment by timiyay** [comment]
Nice one @scalvert, thanks! 🎁 

I hadn't done anything meaningful on it yet - I wasn't sure how useful others would find this particular assertion. Glad to hear it's handy for others too.

**Comment by buschtoens** [comment]
Implemented in #531. Can be closed. ^__^
